cegs_portal/search/templates/search/v1/dna_feature.html

```diff
@@ -350,32 +349,37 @@ <h1>
                         Find Nearby DNA Features
                     </div>
                 </div>
-                <div class="title-separator min-w-full"></div>
-                <fieldset name="facetfield">
-                    <legend class="flex flex-row group font-bold min-w-full" id="facetHeader{{ forloop.counter }}"><div>{{ facet.name }}</div>
-                    </legend>
-                    <div class="flex flex-row flex-wrap gap-1"
-                        aria-labelledby="facetHeader{{ forloop.counter }}">
-                        {% for feature_type in dna_feature_types %}
-                        <div class="checkbox-divider">
-                            <input type="checkbox" id="{{ feature_type }}" name="feature_type"></input>
-                            <label for="{{ feature_type }}">{{ feature_type }}</label>
+                {% for feature, sources, targets, reo in feature_reos %}
```

Comment:
Does this mean that, in the case of multiple features, there are multiple forms? And how are we making sure the results are from the right assembly?

Reply:
Only one form 
![image](https://github.com/ReddyLab/cegs-portal/assets/19657615/6fbcdf1c-4f02-4267-9a38-dc8426bc0619)

Reply:
Ah right, because of the `if forloop.first`. I think my other question still needs to be considered, though.

Reply:
Oh regarding assembly, the default is the most recent. The other(this) screenshot page with both listed isn't accessible.

Reply:
IIRC The old way of doing this resulted in two tables, one for each assembly. I have an idea for this that we can talk about.

Reply:
It's a tricky problem

Reply:
We're changing the page to only show one feature at a time, so this is no longer really an issue!